docs/format/source/format_release_notes.rst

```diff
@@ -1,6 +1,18 @@
 Release Notes
 =============
 
+
+2.0.1 (3/12/10)
+--------------------
+**Change:** ``UnitSeries`` neurodata_type, which stores the assignment of electrophysiologically recorded waveforms to sorted groups that
+represent putatively different sources.
+
+**Reason:**  ``UnitSeries`` acts as a link between ``SpikeEventSeries`` and the ``Units`` table, allowing a user to
+find the recorded waveforms of specific sorted units.
+
+**Specific Changes:** ``UnitSeries`` was added to the ``misc`` module of the schema, and a ``UnitSeries`` class was
+added as a class to ``misc.py``.
```

Comment:
Please add a link to the corresponding PR in PyNWB. This is helpful to allow folks to look at the diff for this specific change.